client/src/modals/ProtectionModal.jsx

```diff
@@ -14,8 +14,12 @@ const ProtectionModal = ({ show, handleClose}) => {
         <hr></hr>
         <div className={ styles.body }>
           <div className={ styles.protectionInfo }>
-            <h3>Protection Plan Name</h3>
-            <h5>from <span>plan provider</span></h5>
+            <h2>{productData.protection_plan.name}</h2>
+            <p>from <span>{productData.protection_plan.provider}</span></p>
+            <div className="stars">
+              <img></img>
```

Comment:
Not sure what this will eventually be used for, but in general i'd keep images as single tags since there's never html content within